### PR TITLE
update osx_attacks with Backdoor.MAC.Eleanor with fixes

### PR DESCRIPTION
**Pull request by PickmanSec**
update based on this blog post:
https://blog.malwarebytes.com/cybercrime/2016/07/new-mac-backdoor-malware-eleanor/

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

File: packs/osx-attacks.conf [hunk 1]
```diff
@@ -237,6 +237,12 @@
       "description": "(https://threatpost.com/mac-adware-osx-pirrit-unleashes-ad-overload-for-now/117273/)",
       "value": "Artifact used by this malware"
     },
+    "Backdoor_MAC_Eleanor": {
+      "query": "select * from launchd where name like 'com.getdropbox.dropbox.integritycheck.plist' or name like 'com.getdropbox.dropbox.timegrabber.plist' or name like 'com.getdropbox.dropbox.usercontent.plist';",
```
Comment (zwass) [inline packs/osx-attacks.conf]: Since there are no wildcards here, let's please use `=` instead of `like`.

Comment (theopolis) [inline packs/osx-attacks.conf]: You could also write:
```sql
SELECT * FROM launchd WHERE name IN (
  'com.getdropbox.dropbox.integritycheck.plist',
  'com.getdropbox.dropbox.timegrabber.plist',
  'com.getdropbox.dropbox.usercontent.plist'
);
```

**Comment by PickmanSec** [comment]
Nice catch, 'like' statement removed.

**Comment by theopolis** [comment]
ok to test

**Comment by zwass** [comment]
Damn you're fast @theopolis!

**Comment by osqueryer** [comment]
Build origin/pr/2226/merge (Job: 2848) is good! :+1:

**Pull request merged**